Branch-SDK/Branch-SDK/BNCCommerceEvent.h

```diff
@@ -1,269 +1,270 @@
 //
 //  BNCCommerceEvent.h
-//  BranchSDK-iOS
+//  Branch-SDK
 //
 //  Created by Edward Smith on 12/14/16.
 //  Copyright (c) 2016 Branch Metrics. All rights reserved.
 //
 
-
 @import Foundation;
 #import "BNCServerRequest.h"
-
+#import "BNCXcode7Support.h"
 
 #pragma mark BNCProductCategory
 
-typedef NSString*const BNCProductCategory;
+typedef NSString*const BNCProductCategory NS_STRING_ENUM;
 
-extern BNCProductCategory BNCProductCategoryAnimalSupplies;
-extern BNCProductCategory BNCProductCategoryApparel;
-extern BNCProductCategory BNCProductCategoryArtsEntertainment;
-extern BNCProductCategory BNCProductCategoryBabyToddler;
-extern BNCProductCategory BNCProductCategoryBusinessIndustrial;
-extern BNCProductCategory BNCProductCategoryCamerasOptics;
-extern BNCProductCategory BNCProductCategoryElectronics;
-extern BNCProductCategory BNCProductCategoryFoodBeverageTobacco;
-extern BNCProductCategory BNCProductCategoryFurniture;
-extern BNCProductCategory BNCProductCategoryHardware;
-extern BNCProductCategory BNCProductCategoryHealthBeauty;
-extern BNCProductCategory BNCProductCategoryHomeGarden;
-extern BNCProductCategory BNCProductCategoryLuggageBags;
-extern BNCProductCategory BNCProductCategoryMature;
-extern BNCProductCategory BNCProductCategoryMedia;
-extern BNCProductCategory BNCProductCategoryOfficeSupplies;
-extern BNCProductCategory BNCProductCategoryReligious;
-extern BNCProductCategory BNCProductCategorySoftware;
-extern BNCProductCategory BNCProductCategorySportingGoods;
-extern BNCProductCategory BNCProductCategoryToysGames;
-extern BNCProductCategory BNCProductCategoryVehiclesParts;
+FOUNDATION_EXPORT  BNCProductCategory _Nonnull BNCProductCategoryAnimalSupplies;
+FOUNDATION_EXPORT  BNCProductCategory _Nonnull BNCProductCategoryApparel;
+FOUNDATION_EXPORT  BNCProductCategory _Nonnull BNCProductCategoryArtsEntertainment;
+FOUNDATION_EXPORT  BNCProductCategory _Nonnull BNCProductCategoryBabyToddler;
+FOUNDATION_EXPORT  BNCProductCategory _Nonnull BNCProductCategoryBusinessIndustrial;
+FOUNDATION_EXPORT  BNCProductCategory _Nonnull BNCProductCategoryCamerasOptics;
+FOUNDATION_EXPORT  BNCProductCategory _Nonnull BNCProductCategoryElectronics;
+FOUNDATION_EXPORT  BNCProductCategory _Nonnull BNCProductCategoryFoodBeverageTobacco;
```

Comment:
could we make this FoodBeveragesTobaco? just for consistency everywhere? (i think these categories are from google)

Reply:
The `BNCProductCategoryFoodBeverageTobacco` is an existing definition so changing it will break people's compiles.

Reply:
urgh, ok